docs/src/man/getting_started.md

```diff
@@ -1053,15 +1132,18 @@ The CSV.jl functions are not loaded automatically and must be imported into the
 using CSV
 ```
 
-A dataset can now be read from a CSV file at path `input` using
-```julia
-DataFrame(CSV.File(input))
-```
+You can read a .csv file containing your dataset at file pathname _fpath_ using (Windows example):
 
-A `DataFrame` can be written to a CSV file at path `output` using
-```julia
-df = DataFrame(x = 1, y = 2)
-CSV.write(output, df)
 ```
+fpath = "C:\\Users\\biff\\Documents\\myData\\mydataset.csv"
```

Comment:
use `\\` instead of `/` this will work both on Windows and Linux. If you write such an example then please comment that this command will not work and this is just an example.